dpsim/examples/cxx/cim_graphviz/CMakeLists.txt

```diff
@@ -1,4 +1,4 @@
 if(WITH_GRAPHVIZ AND WITH_CIM)
```

Comment:
Shouldn't `WITH_CIM` then be removed here, if `CIMPP_LIBRARIES` not anymore considered below?

Reply:
I think we should probably also link against cimpp here. I added it as a library dependency.